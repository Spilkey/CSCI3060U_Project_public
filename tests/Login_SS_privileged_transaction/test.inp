login
user04
create
user01
AA
logout
exit