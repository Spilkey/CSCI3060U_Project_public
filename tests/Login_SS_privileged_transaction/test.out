01 user01          AA 000000000
00 user04          SS 000000000